web/config/environment.js

```diff
@@ -49,6 +49,13 @@ module.exports = function (environment) {
       apiKey: getEnv("ALGOLIA_SEARCH_API_KEY"),
     },
 
+    flashMessageDefaults: {
+      timeout: 5000,
+      extendedTimeout: 1000,
+      type: "success",
+      types: ["critical", "success"],
+    },
```

Comment:
What's the use case(s) you're thinking for making these configurable per environment?

Reply:
No plans for environment-specific config yet (though I could see using short timeouts if `environment === "testing"`). I'm just using `environment.js` to be in accordance with the [addon configuration](https://github.com/adopted-ember-addons/ember-cli-flash#service-defaults).